model/map/src/main/java/org/keycloak/models/map/storage/chm/MapFieldPredicates.java

```diff
@@ -120,9 +120,11 @@ public class MapFieldPredicates {
         put(CLIENT_SCOPE_PREDICATES, ClientScopeModel.SearchableFields.NAME,      MapClientScopeEntity::getName);
 
         put(GROUP_PREDICATES, GroupModel.SearchableFields.REALM_ID,               MapGroupEntity::getRealmId);
+        put(GROUP_PREDICATES, GroupModel.SearchableFields.ID,                     MapGroupEntity::getId);
```

Comment:
This is not necessary, The `ID` is added already by calling `basePredicates(GroupModel.SearchableFields.ID);` at line 87.

Reply:
removed